src/sage/graphs/generators/families.py

```diff
@@ -1711,6 +1711,13 @@ def GeneralizedPetersenGraph(n, k):
         sage: g.is_bipartite()
         True
 
+    TESTS::
```

Comment:
only one colon in this line

Reply:
That's confusing, because I copied the syntax from elsewhere in the file. E.g. in the definition of `MathonPseudocyclicMergingGraph` (line 3829) SEEALSO and TEST both have double-colons. There are ~20 other places with `TEST::`. Are these also docstring bugs that I should fix while I'm working on this file?

Reply:
double colon means that it is followed by and indented block. Here it is not because you have another line with the same indent, itself ending with a double colon and followed by its indented block. So : just put one single colon at the end of TESTS on the one line that I designated